src/parse/properties/css_property_parser_gen.c

```diff
@@ -609,45 +604,61 @@ int main(int argc, char **argv)
 		} else if (curlist == &base && strcmp(rkv->key, "NUMBER") == 0) {
 			if (rkv->val[0] == '(') {
 				curlist = &NUMBER;
+				free(rkv); /* Free opening parenthesis */
 			} else if (rkv->val[0] == ')') {
 				curlist = &base;
+				free(rkv); /* Free closing parenthesis */
 			} else {
 				NUMBER.item[NUMBER.count++] = rkv;
 			}
 			only_ident = false;
 		} else if (strcmp(rkv->key, "IDENT") == 0) {
 			if (rkv->val[0] == '(') {
 				curlist = &IDENT;
+				free(rkv); /* Free opening parenthesis */
 			} else if (rkv->val[0] == ')') {
 				curlist = &base;
+				free(rkv); /* Free closing parenthesis */
 			} else if (strcmp(rkv->val, str_INHERIT) == 0) {
 				IDENT.item[IDENT.count++] = &ident_inherit;
+				free(rkv); /* Free rkv, use static */
 			} else if (strcmp(rkv->val, str_INITIAL) == 0) {
 				IDENT.item[IDENT.count++] = &ident_initial;
+				free(rkv); /* Free rkv, use static */
 			} else if (strcmp(rkv->val, str_REVERT) == 0) {
 				IDENT.item[IDENT.count++] = &ident_revert;
+				free(rkv); /* Free rkv, use static */
 			} else if (strcmp(rkv->val, str_UNSET) == 0) {
 				IDENT.item[IDENT.count++] = &ident_unset;
+				free(rkv); /* Free rkv, use static */
+			} else {
+				IDENT.item[IDENT.count++] = rkv;
```

Comment:
This part looks like a change of behavior, but I don't have time to look into whether its correct today. Previously it would have been dropped, rather than added to the array.